config/v3_5_experimental/schema/ignition.json

```diff
@@ -236,6 +236,9 @@
               "items": {
                 "type": "string"
               }
+            },
+            "assemble": {
```

Comment:
Since you added a struct field you need to add a translation for it in translations.